tests/sequence_break/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -79,6 +90,52 @@ for (
     : function() {
         return 0;
       });
+if (
+  aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName
+) {
+}
+while (
+  aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName
+) {}
+switch (
+  aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName
+) {
+}
+with (aLongIdentifierName,
+aLongIdentifierName,
+aLongIdentifierName,
+aLongIdentifierName)
+  console.log(aLongIdentifierName);
+(function() {
+  throw aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName,
+  aLongIdentifierName;
```

Comment:
looks ugly for me. I think `ThrowStatement` should be formatted like `ReturnStatement`.

Reply:
Me too.